local_db.py

```diff
@@ -436,8 +441,8 @@
         'root@test.org.uk',
         ('$pbkdf2-sha256$29000$UAqBcA6hVGrtvbd2LkW'
          'odQ$4nNngNTkEn0d3WzDG31gHKRQ2sVvnJuLudwoynT137Y'),
-        countries + ['jordan', 'madagascar', 'somalia', 'meerkat'],
-        ['root' for c in countries] + ['root', 'root', 'root', 'root'],
+        countries + ['jordan', 'madagascar', 'somalia', 'meerkat', 'somaliland'],
+        ['root' for c in countries] + ['root', 'root', 'root', 'root', 'root'],
```

Comment:
That's very strange construction :)
I'd create a separate list with `foobar = countries + ['jordan', 'madagascar', 'somalia', 'meerkat', 'somaliland']` and then you can have ['root'] * len(foobar).